core/src/main/java/org/bitcoinj/core/BlockHeader.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2011 Google Inc.
+ * Copyright 2014 Andreas Schildbach
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bitcoinj.core;
+
+import javax.annotation.*;
+
+/**
+ * <p>A block is a group of transactions, and is one of the fundamental data structures of the Bitcoin system.
```

Comment:
It's a child of HeadersMessage, but yes, it's not used as a stand-alone message. Not quite sure what to do with that, unless we're going to start reshuffling the Message heirarchy in a major way (i.e. make Message and ChildMessage interfaces, rename Message to AbstractMessage)